test/test-state.js

```diff
@@ -103,6 +103,23 @@ describe("State", () => {
   })
 
   it("allows plugins to append transactions", () => {
+    let tr
+    let state = EditorState.create({plugins: [
+      new Plugin({
+        appendTransaction: (trs, oldState, newState) =>
+          newState.tr.insertText("Y")
+      }),
+      new Plugin({
+        appendTransaction: trs => {
+          ist(tr, trs[1].getMeta("appendedTransaction"))
+        }
+      })
+    ], schema})
+    tr = state.tr.insertText("X")
+    state.applyTransaction(tr)
+  })
+
+  it("stores a reference to a rootTransaction for appended transactions", () => {
```

Comment:
The test below doesn't seem to be testing what the string here describes. Could it be that you flipped the test descriptions?